fedimint-api/src/task.rs

```diff
@@ -52,12 +161,13 @@ mod imp {
 
     use super::*;
 
-    pub fn spawn<F>(future: F)
+    pub(crate) fn spawn<F>(future: F) -> Option<JoinHandle<()>>
     where
         // No Send needed on wasm
         F: Future<Output = ()> + 'static,
     {
-        wasm_bindgen_futures::spawn_local(future)
+        wasm_bindgen_futures::spawn_local(future);
```

Comment:
For `wasm` no `JoinHandle` is returned, so we will just not be able to wait for the tasks to shut down, I guess. :shrug: